Library/Formula/predictionio.rb

```diff
@@ -1,8 +1,8 @@
 class Predictionio < Formula
   desc "Source machine learning server"
   homepage "https://prediction.io/"
-  url "http://download.prediction.io/PredictionIO-0.9.2.tar.gz"
-  sha256 "e4da196a1d67f545d6667f8368a7c6a64ff5c6de5135b085a12e9251b6076991"
+  url "https://d8k1yxp8elc6b.cloudfront.net/PredictionIO-0.9.5.tar.gz"
```

Comment:
I'd be tempted to keep this at the `http://download.prediction.io` download URL. Random looking Cloudfront URLs can tend to feel a bit shady, even when that isn't the case.